### ISSUE TITLE
Incorrectly removing "requires" from package lock

### ISSUE THREAD

**Issue by dragomir-enachi-ibm** [rogeriochaves/npm-force-resolutions#21]
**Issue:** Incorrectly removing "requires" from package lock

**Description:**
After applying npx force resolutions,"requires" section is being removed, together with "**integrity**" and "**resolved**"

![image](https://user-images.githubusercontent.com/49532417/105426457-1502c780-5c43-11eb-85de-fa85e011378a.png)

**To replicate:**

1. make changes to package JSON:

`"dependencies": {
        "sfdx-cli": "^7.84.2"
    },
    "devDependencies": {},
    "resolutions": {
        "axios": "0.21.1"
    }`
2. Run npm install
3. Run npx npm-force-resolutions

**Result:** As per screenshot above.
Removed lines:
```
-          "version": "0.21.1",
-          "resolved": "https://registry.npmjs.org/axios/-/axios-0.21.1.tgz",
-          "integrity": "sha512-dKQiRHxGD9PPRIUNIWvZhPTPpl1rf/OxTYKsqKUDjBwYylTvV7SjSHJb9ratfyzM6wCdLCOYLzs73qpg5c4iGA==",
-          "requires": {
-            "follow-redirects": "^1.10.0"
-          }
```
Added lines:
```"version": "0.21.1"```

in axios dependency.

Running NPM ci after that would not install required follow-redirects

Can't rely on the script anymore 😞

**Comment by rogeriochaves** [comment]
Fixed by aed7845ce834e18e0bc5773e31d0fd7abd6ed675, I will create a new release soon

**Comment by rogeriochaves** [comment]
Additionally, I pushed a fix to now include the `resolved` and `integrity` keys as well on b006bb3d2ae2160f5620ef86627d882eeffefb28

Released on v0.0.4, enjoy!
